src/mcp/tools/crashlytics/index.ts

```diff
@@ -2,14 +2,7 @@ import type { ServerTool } from "../../tool";
 import { create_note, list_notes, delete_note } from "./notes";
 import { get_issue, update_issue } from "./issues";
 import { list_events, batch_get_events } from "./events";
-import {
-  get_top_issues,
-  get_top_variants,
-  get_top_versions,
-  get_top_apple_devices,
-  get_top_operating_systems,
-  get_top_android_devices,
-} from "./reports";
+import { get_report } from "./reports";
```

Comment:
When we remove MCP tools, what happens to the history of the MCP calls made by the agents in the IDE? Do you know if those caches would be invalidated? 

Not a blocker - but was curious what happens in those cases.